styles.css

```
@@ -95,6 +111,18 @@ h4 {
   animation: fadeIn 7.8s;
 }
 
+.scroll-down {
+  max-height: 10%;
+  flex-basis: 5%;
+  flex-shrink: 0.5;
+  animation: fadeIn 9.5s ease-out;
```

It feels too bright to me, let's make it grey? :)
```suggestion
  filter: contrast(0%);
  animation: fadeIn 9.5s ease-out;
```

---

Done. looks better.